css/style1.css

```diff
@@ -27,22 +27,15 @@ body {
   }
 }
 
-.faculty-container {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-
 #main-title {
-  color: #FFDE7A;
-  font-family: 'Patua One', monospace;
+  color: #ffde7a;
+  font-family: "Patua One", monospace;
```

Comment:
Again, any particular reason to change here? [Google's guidelines](https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks), for example, are to use _single_ quotes in CSS. (Although that decision seems pretty arbitrary, and I don't know why they recommend double quotes for HTML and single for CSS.)

Reply:
I just used the "Format Document" option in VS Code which since VS Code doesn't have a built-in formatter for CSS, used the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension I have installed. I suppose it just used the default settings combined with the `.editorconfig`.

Reply:
I believe that everywhere else in future labs and the iteration template SCSS we use double quotes